src/Stack/Types/PackageDump.hs

```diff
@@ -31,4 +31,6 @@ data InstalledCacheEntry = InstalledCacheEntry
 instance Store InstalledCacheEntry
 
 installedCacheVC :: VersionConfig InstalledCacheInner
-installedCacheVC = storeVersionConfig "installed-v1" "GGyaE6qY9FOqeWtozuadKqS7_QM="
+-- FIXME would be really nice to avoid changing this hash...
+--installedCacheVC = storeVersionConfig "installed-v1" "GGyaE6qY9FOqeWtozuadKqS7_QM="
```

Comment:
> That's unfortunate, I was hoping there was some way to add backwards-compatibility support using the old serialization format.

That is possible, but of course the datatypes will need to be duplicated.  Some logic would need to be implemented to handle recognizing the old tag and doing the datatype migration.